### PR TITLE
Add blog post about moving Rust's CI to GitHub Actions

### PR DESCRIPTION
**Pull request by pietroalbini**
The post should be merged tomorrow.

r? @rust-lang/infra

### CODE REVIEW

File: posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Making the CI setup faster is a permanent goal of the Infrastructure Team, and
+GitHub Actions provided us with a great opportunity to improve landing time:
+GitHub offered to sponsor a fully managed, private pool of 8-core VMs to run
+our builds on, which is a big improvement compared to the 2-core VMs we were
+previously using. GitHub Actions also provides most of the features we loved
+about Azure Pipelines while being integrated with GitHub’s permissions and UI,
+which made the switch even more fruitful.
+
+As of July 22th, all the CI builds for the [rust-lang/rust] repository except
```
Comment (ehuss) [inline posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md:30]:
```suggestion
As of July 22nd, all the CI builds for the [rust-lang/rust] repository except
```

?

File: posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Rust's CI is moving to GitHub Actions"
+author: Pietro Albini
+team: the infrastructure team <https://www.rust-lang.org/governance/teams/operations#infra>
+---
+
+The Rust Infrastructure Team is happy to announce that, as part of the
+[evaluation we started last year][eval], most of Rust’s CI is moving to GitHub
+Actions! While we don’t expect the change to have any impact to our users, the
```
Comment (tshepang) [inline posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md:10]:
```suggestion
Actions! While we don’t expect the change to have any impact on our users, the
```

File: posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+switch will considerably improve the experience for compiler contributors.
+
+One of the major pain points for compiler contributors over the past few years
+has been waiting for PRs to be merged. We value having an always-green master
+branch, and to ensure that we test and merge just one PR at a time, with the
```
Comment (tshepang) [inline posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md:15]:
```suggestion
branch, and to ensure that, we test and merge just one PR at a time, with the
```

File: posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+other approved ones [waiting in the queue][homu]. Our CI is extensive too, with
+57 machines building and testing the compiler across all the platforms we
+support. On our previous system, each of those builders takes between three to
```
Comment (tshepang) [inline posts/inside-rust/2020-07-23-rust-ci-is-moving-to-github-actions.md:18]:
```suggestion
support. On our previous system, each of those builders took between three to
```

**Pull request merged**